content/open-source/apko/reference.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "YAML Reference"
+type: "article"
+description: "apko YAML Reference"
+date: 2022-10-10T11:07:52+02:00
+lastmod: 2022-10-10T11:07:52+02:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "apko"
+weight: 150
+toc: true
+---
+
+In this page you can find a reference for apko's YAML specification.
+
+## contents
```

Comment:
```suggestion
## Contents
```

Reply:
In this case, `contents` is the yaml node name all lowercase, should I maybe format it differently using backticks?

Reply:
I tried using ## `contents` but I think it didn't look so good. I will keep it like this for now, maybe we think of a better organizational structure for these in order to have a nicely built TOC